pcsx2-qt/Settings/GraphicsSettingsWidget.cpp

```diff
@@ -320,12 +320,18 @@ GraphicsSettingsWidget::GraphicsSettingsWidget(SettingsDialog* dialog, QWidget*
 			m_ui.videoCaptureContainer->addItem(name.toUpper(), name);
 		}
 
+		SettingWidgetBinder::BindWidgetToFolderSetting(sif, m_ui.videoDumpingDirectory, m_ui.videoDumpingDirectoryBrowse, m_ui.videoDumpingDirectoryOpen, m_ui.videoDumpingDirectoryReset,
+			"Folders", "Videos", Path::Combine(EmuFolders::DataRoot, "videos"));
+
 		SettingWidgetBinder::BindWidgetToStringSetting(sif, m_ui.videoCaptureContainer, "EmuCore/GS", "VideoCaptureContainer");
 		connect(m_ui.videoCaptureContainer, &QComboBox::currentIndexChanged, this, &GraphicsSettingsWidget::onVideoCaptureContainerChanged);
 
 		SettingWidgetBinder::BindWidgetToIntSetting(
 			sif, m_ui.videoCaptureBitrate, "EmuCore/GS", "VideoCaptureBitrate", Pcsx2Config::GSOptions::DEFAULT_VIDEO_CAPTURE_BITRATE);
 
+		SettingWidgetBinder::BindWidgetToBoolSetting(sif, m_ui.enableVideoCaptureArguments, "EmuCore/GS", "EnableVideoCaptureParameters", false);
```

Comment:
Should we disable the capture arguments if the enable manual option isn't changed? (i.e. connect to `stateChanged`, and enable/disable `videoCaptureArguments`). See audio options for an example.

Reply:
That is what I was wondering whether to do.
Done.